src/communication/pipe.hpp

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef PIPE_HPP_
+#define PIPE_HPP_
+
+#include <aos/common/tools/string.hpp>
+
+#include "transport.hpp"
+
+namespace aos::zephyr::communication {
+
+class Transport : public TransportItf {
+public:
+    static constexpr auto cXSReadPath  = CONFIG_AOS_CHAN_TX_PATH;
```

Comment:
XS stands for xen strore, so not relevant for pipe. Could be just `cReadPath`, `cWritePath`

Reply:
for more general case i prefer use `cReadPath ` and `cWritePath ` than in code will be easy to use `Transport::cReadPath ` will  be more general

Reply:
Done